src/pages/ttt.svelte

```diff
@@ -14,8 +12,8 @@
 	$: totalCount = turnCount + pcTurnCount;
 	let success = false;
 	let blocked = false;
-	$: winner === "O" && $count++;
-	$: winner === "X" && $count--;
+	$: winner === "O" && addPoint();  //move to player service
```

Comment:
was this a leftover TODO:?

Reply:
Ahhh, yes. I already did that so will delete the comment.